src/ostorlab/runtimes/local/models/models.py

```diff
@@ -127,6 +127,9 @@ class Scan(Base):
     asset = sqlalchemy.Column(sqlalchemy.String(255))
     created_time = sqlalchemy.Column(sqlalchemy.DateTime)
     progress = sqlalchemy.Column(sqlalchemy.Enum(ScanProgress))
+    agent_group_id = sqlalchemy.Column(
+        sqlalchemy.Integer, sqlalchemy.ForeignKey("agent_group.id")
+    )
```

Comment:
Is this nullable, waht will happen for the existing scans?

Reply:
Iti is nullable
![image](https://github.com/Ostorlab/oxo/assets/144013278/de07c9ac-0c4f-4d4c-aa5a-c34bcddfd0db)